### PR TITLE
HARMONY 718: Basic repo setup

### PR DESCRIPTION
**Pull request by rwat**
Please see the following card: https://bugs.earthdata.nasa.gov/browse/HARMONY-718

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+
+1. Fork this repository to your personal account.
+2. Create a branch and make your changes.
+3. Test the changes locally/in your personal fork.
+4. Submit a pull request to open a discussion about your proposed changes.
+5. The maintainers will talk with you about it and decide to merge or request additional changes.
+
+## Commits
+
+Our ticketing and CI/CD tools are configured to sync statuses amongst each other. Commits play an important role in this process. Please start all commits with the Harmony ticket number associated with your feature, task, or bug. All commit messages should follow the format "harmony-XXXX: [Your commit message here]"
```
Comment (chris-durbin) [inline CONTRIBUTING.md:17]: minor nit - looks like we usually use all caps in our commit messages to match JIRA - HARMONY-XXX

File: docs/_build/html/.buildinfo [hunk 2]
```diff
@@ -0,0 +1,4 @@
+# Sphinx build info version 1
```
Comment (chris-durbin) [inline docs/_build/html/.buildinfo:1]: I don't really know anything about Sphinx. Is this file meant to be tracked by git?

File: docs/conf.py [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+
+
+# -- Project information -----------------------------------------------------
+
+project = 'harmony-py'
+copyright = '2021, NASA Harmony Project'
+author = 'NASA Harmony Project'
+
+# The full version, including alpha/beta/rc tags
+release = '0.0.1'
```
Comment (chris-durbin) [inline docs/conf.py:25]: Any guidance on how we'll handle release versioning? Are we going to control versions manually or have them assigned from Bamboo?

**Review by chris-durbin — COMMENTED** [review]
I wasn't clear if the _build directory should be managed in git. It looks like those are generated files by running the sphinx build. If so it seems like they should not be committed. Otherwise looks good to me.

File: README.md [hunk 4]
```diff
@@ -0,0 +1,7 @@
+# harmony-py
+
+`harmony-py` is a Python library for integrating with NASA's [Harmony](https://harmony.earthdata.nasa.gov/).
+
+`harmony-py` is an alternative to [Harmony's RESTful API](https://harmony.earthdata.nasa.gov/docs/api/), handles NASA [Earthdata Login (EDL)](https://urs.earthdata.nasa.gov/home) authentication and optionally integrates with the [CMR Python Wrapper](https://github.com/nasa/eo-metadata-tools) by accepting collection results as a request parameter. It's convenient for scientists who wish to use Harmony from Jupyter notebooks as well as machine-to-machine communication for larger Python applications.
+
+`harmony-py` is a work-in-progress, is not feature complete, and should only be used if you would like to test `harmony-py` functionality.
```
Comment (chris-durbin) [inline README.md:7]: Other potentially useful instructions to add to the README:

1. How to install this library in your project
2. How to run the linter (should we add something to the makefile so we can run `make lint`?)
3. How to run the tests (maybe include one dummy test?)

Comment (chris-durbin) [reply README.md:7]: I forgot - also how to generate the docs.

File: docs/conf.py (hunk 3 above)
Comment (rwat) [reply docs/conf.py:25]: I haven't heard anything about that.

File: README.md (hunk 4 above)
Comment (rwat) [reply README.md]: Good improvements - added.

File: docs/conf.py (hunk 3 above)
Comment (kbeamnsidc) [reply docs/conf.py:25]: I'd like to use poetry or something like it which will do semantic versioning. You can then do `poetry version patch`, `poetry version minor`, `poetry version premajor`, ... And also do GitHub releases when we want to release a version.

**Review by chris-durbin — APPROVED** [review]

File: docs/conf.py (hunk 3 above)
Comment (rwat) [reply docs/conf.py:25]: Just so there's record of it here, we talked about including poetry in a future PR.

**Review by kbeamnsidc — APPROVED** [review]

**Pull request merged**